### PR TITLE
Refactor the plus/mius to use GOV UK accordion icons

### PR DESCRIPTION
**Pull request by RobinKnipe**
As a [frankly failed] experiment, the plus and minus icons used to mark the CID ID detail drilldowns have been changed to the standard "accordian" icons from the GOV UK frontend toolkit.

Like so...

image

### CODE REVIEW

File: gulp/config.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ export default {
     dest: 'build/images'
   },
 
+  govuk_images: {
+    src: 'node_modules/govuk_frontend_toolkit/images/accordian-arrow.png',
```
Comment (chrisns) [inline gulp/config.js]: you should be able to do `path.dirname(require.resolve("govuk_frontend_toolkit"))` to get a safer path to the `govuk_frontend_toolkit`

Comment (chrisns) [inline gulp/config.js]: actually, better yet: `require.resolve("govuk_frontend_toolkit/images/accordian-arrow.png")`

File: app/styles/main.scss [hunk 2]
```diff
@@ -151,19 +151,33 @@ footer {
...
+      background: url('../images/accordian-arrow.png') no-repeat 0 -5px;
+      background-size: 30px;
+    }
+    tr.close > td:first-child:before {
+      content: "";
+      float: left;
+      width: 45px;
+      height: 20px;
+      background: url('../images/accordian-arrow.png') no-repeat 0 -35px;
+      background-size: 30px;
```
Comment (RobinKnipe) [inline app/styles/main.scss]: Yes. The above is sick-in-mouth horrible CSS, hence the comment on the commit, but it was the only way I could make use of the stupid 2-icons-in-1-stupid-png nonsense that is the GOV UK frontend toolkit.

Comment (chrisns) [inline app/styles/main.scss]: presumably they provide sass that we could use instead of essentially copying the 'nonsense' sprite?

**Pull request merged**